go.mod

```diff
@@ -20,7 +20,7 @@ require (
 	github.com/giantswarm/helmclient v0.2.0
 	github.com/giantswarm/ipam v0.2.0
 	github.com/giantswarm/k8sclient v0.2.0
-	github.com/giantswarm/k8scloudconfig/v6 v6.0.1
+	github.com/giantswarm/k8scloudconfig/v6 v6.0.2-0.20200331075245-f7153f38c6d0
```

Comment:
Looks like this needs a release of v6.0.2 before merging.

Reply:
released and changed